pkg/rook/common/logs.go

```diff
@@ -8,11 +8,20 @@ import (
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 )
 
-var ValidCephSubSystems = []string{"osd", "mds", "mon", "mgr", "auth"}
+var (
+	ValidCephSubSystems = []string{"default", "lockdep", "context", "crush", "mds", "mds_balancer", "mds_log", "mds_log_expire",
+		"mds_migrator", "buffer", "timer", "filer", "striper", "objecter", "rados", "rbd", "rbd_mirror", "rbd_replay", "rbd_pwl", "journaler",
+		"objectcacher", "immutable_obj_cache", "client", "osd", "optracker", "objclass", "filestore", "journal", "ms", "mon", "monc", "paxos",
+		"tp", "auth", "crypto", "finisher", "reserver", "heartbeatmap", "perfcounter", "rgw", "rgw_sync", "rwg_datacache", "rgw_access",
+		"rgw_dbstore", "javaclient", "asok", "throttle", "refs", "compressor", "bluestore", "bluefs", "bdev", "kstore", "rocksdb", "fuse", "mgr",
+		"mgrc", "dpdk", "eventtrace", "prioritycache", "test", "cephfs_mirror", "cepgsqlite", "seastore", "seastore_onode", "seastore_odata",
+		"seastore_ompap", "seastore_tm", "seastore_t", "seastore_cleaner", "seastore_epm", "seastore_lba", "seastore_fixedkv_tree", "seastore_cache",
+		"seastore_journal", "seastore_device", "seastore_backref", "alienstore", "mclock", "cyanstore", "ceph_exporter", "memstore", "trace"}
+)
 
 func SetLogLevel(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, storageClusterNamespace string, args []string) {
```

Comment:
Let's use names for all params for readability
```suggestion
func SetLogLevel(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, storageClusterNamespace string, component, subsystem, level string) {
```

Reply:
updated